### ISSUE TITLE
Add arm64e slice to Firebase Analytics

### ISSUE THREAD

**Issue by doganov** [firebase/firebase-ios-sdk#4110]
A full year has passed since arm64e was announced. The arm64e support in Xcode 11 is still considered experimental, but Apple advises us to test our apps in advance in order to prepare in for the moment when arm64e becomes a required architecture. App authors still can't do that today if their app depends on Firebase Analytics.

This also applies to all binary-only dependencies that Firebase Analytics itself depends on.

### [READ] Step 1: Are you in the right place?

  * For issues or feature requests related to __the code in this repository__
    file a Github issue.
    * If this is a __feature request__ make sure the issue title starts with "FR:".
  * For general technical questions, post a question on [StackOverflow](http://stackoverflow.com/)
    with the firebase tag.
  * For general Firebase discussion, use the [firebase-talk](https://groups.google.com/forum/#!forum/firebase-talk)
    google group.
  * For help troubleshooting your application that does not fall under one
    of the above categories, reach out to the personalized
    [Firebase support channel](https://firebase.google.com/support/).

### [REQUIRED] Step 2: Describe your environment

  * Xcode version: 11.1
  * Firebase SDK version: 6.10.0 installed via Cocoapods
  * Firebase Component: Analytics (Auth, Core, Database, Firestore, Messaging, Storage, etc)
  * Component version: 6.10.0 (Core), 6.1.3 (Analytics)

#### Steps to reproduce:

When arm64e is enabled as an architecture, Xcode 11.1 fails to link the target app because there is missing arm64e slice in Firebase Analytics (and perhaps some of its dependencies):

```
Undefined symbols for architecture arm64e:
  "_OBJC_CLASS_$_FIRAnalytics", referenced from:
      objc-class-ref in IosAnalytics.o
     (maybe you meant: _OBJC_CLASS_$_FIRAnalyticsConfiguration)
ld: symbol(s) not found for architecture arm64e
```

**Comment by paulb777** [comment]
Implementation is done.  See #4342.  

The release will go out the next time Analytics releases with Firebase - which is still TBD.

**Comment by paulb777** [comment]
Now planned for the next release.

**Comment by paulb777** [comment]
Fix released with Firebase 6.14.0
